### ISSUE TITLE
Unable to convert from leder-parser Ledger with multiple currency

### ISSUE THREAD

**Issue by hawkpro-1** [marek-g/rust-ledger-utils#4]
With version `0.3.0` and above. I am not able to convert from ledger_parser to simplified ledger when it contains multiple currency. Is there any other way as other modules such as monthlyreports only works simplified ledger

This is the error and ledger file that I use to reproduce the error
```
let ledger = ledger_parser::parse(&file_string);
let mut simple_ledger = SimpleLedger::try_from(ledger);
```
```
2020-02-01 Buy ADA
    assets:cc:ada          2000 ADA @ $0.02
    assets:bank:checking                   $-40
```
with this error
```
Error: UnbalancedTransaction(Transaction { comment: None, date: 2020-02-01, effective_date: None, status: None, code: None, description: "Buy ADA", postings: [Posting { account: "assets:cc:ada", reality: Real, amount: Some(PostingAmount { amount: Amount { quantity: 2000, commodity: Commodity { name: "ADA", position: Right } }, lot_price: None, price: Some(Unit(Amount { quantity: 0.02, commodity: Commodity { name: "$", position: Left } })) }), balance: None, status: None, comment: None }, Posting { account: "assets:bank:checking", reality: Real, amount: Some(PostingAmount { amount: Amount { quantity: -40, commodity: Commodity { name: "$", position: Left } }, lot_price: None, price: None }), balance: None, status: None, comment: None }] })
```

Thank :)

**Comment by marek-g** [comment]
Yes, thank you for reporting. I definitely want to fix it soon as it also affects my use cases.

The bug is here since introduction of SimpleLedger (which is useful to calculate missing amounts, which were not supported on reports before, but it fails with multiple currencies).

**Comment by marek-g** [comment]
The issue is fixed  in version 0.4.1 now, but missing amounts during commodity exchanges are not supported yet.

New issue: https://github.com/marek-g/rust-ledger-utils/issues/5
